examples/js/loaders/GLTFLoader.js

```diff
@@ -793,7 +793,7 @@ THREE.GLTFLoader = ( function () {
 
 					}
 
-					textureLoader.crossOrigin = options.crossOrigin || false;
+					if ( options.crossOrigin !== undefined ) textureLoader.setCrossOrigin( options.crossOrigin );
```

Comment:
I think you can just do `textureLoader.setCrossOrigin( options.crossOrigin )`. Is going to be reset to `undefined` anyway.